CRM/Custommosaico/Plugin/FontLoaderPlugin.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+class CRM_Custommosaico_Plugin_FontLoaderPlugin {
```

Comment:
since the folder is already named `plugin` and it is part of the class name already, you don't need to mention the word again, so you can just rename it to: 

`CRM_Custommosaico_Plugin_FontLoader`